intro.txt

```diff
@@ -154,6 +167,40 @@ following procedure can be used:
 
 @endlatexonly
 
+@latexonly
```

Comment:
Technically, should clean this up.  But I guess it's ok if we let it go.  This is the intro to the API guide.